pegasus/sites.v3/ar.code.org/views/top_quotes.haml

```diff
@@ -13,8 +13,9 @@
       %p{:style=>'font-size: 13px;line-height: 18px;'}
         =value[:quote_t]
 
--#links{:style=>'clear:both;'}
-  = view :share_buttons, facebook:facebook, twitter:twitter)
-  .all-quotes-link
-    %a{:href=>'http://code.org/quotes'}
-      Mirá todas las citas (en inglés)
\ No newline at end of file
+-#
+  links{:style=>'clear:both;'}
```

Comment:
Yeah, we hide them on this site. My future PR removes it.

https://github.com/code-dot-org/code-dot-org/pull/7393